main.css

```diff
@@ -67,5 +67,5 @@ h1{
   background-color: rgb(100,100,100);
   padding: 0.5em;
   margin: 0.5em;
-  cursor: default;
+  cursor: pointer;
```

Comment:
yeah.. i should change that label bit. 
and i know. i just find the hand ugly. maybe i'll add a hover effect later.